assignments/prototypes.js

```diff
@@ -32,7 +52,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+ function Humanoid(attributes){
+  this.team = attributes.team;
+  this.weapons = attributes.weapons;
+  this.language = attributes.language;
+  this.healthPoints = attributes.healthPoints;
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+ }
+
+ Humanoid.prototype.greet = function() {
+   return (`${this.name} offers a greeting in ${this.language}.`)
+ }
+
```

Comment:
And here you need to again set up the inheritance from the parent and grandparents prototypes so you will need to 

```js
Humanoid.prototype = Object.Create(CharacterStats.prototype)
```

so You get all those functions from the parent and grandparent prototype, before adding new ones for the humanoid that they don't get.